### DISCUSSION TITLE
installed Analyst VM, but not seeing directories....

### DISCUSSION THREAD

**Discussion by scada-cyber** [Security-Onion-Solutions/securityonion#10547] [General]
Hello,
I recently created an Analyst VM using securityonion-2.3.230-20230417.iso
I'm trying to create some NIDS local rules as mentioned here:
https://docs.securityonion.net/en/2.3/local-rules.html

When I try changing directories to:
cd /opt/so/saltstack/local/salt/idstools/

It says that directory doesn't exist....

I'm only able to change directory to:
cd /opt/so/saltstack/local/

Once I'm in /opt/so/saltstack/local/ there's nothing in that directory, it's blank.

I also noticed "sudo so-status" doesn't work either, and other "sudo so-" commands don't work? Not sure what's going on.

Am I doing something wrong? I just want to add some custom Suricata rules...
please help!

**Comment by scada-cyber** [comment]
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/52265672/c635c273-ce6b-41b1-84b2-ef8dca85c768)

**Reply by robbiemarshall** [reply]
You should be using a Standalone install of the VM instead of an Analyst VM if you're wanting to create custom NIDS rules.

**Comment by scada-cyber** [comment]
Thank you!
I will install the standalone and try it out!
